openpype/lib/attribute_definitions.py

```diff
@@ -443,7 +445,10 @@ def __init__(self, key, items, default=None, **kwargs):
 
         items = self.prepare_enum_items(items)
         item_values = [item["value"] for item in items]
-        if default not in item_values:
+        if multiselection and default is None:
+            default = []
+
+        if not multiselection and default not in item_values:
             for value in item_values:
                 default = value
                 break
```

Comment:
Seems like this was intended to just get the first value?
```suggestion
            default = next(iter(item_values), None)
```